portfolio/src/main/webapp/education.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Education & Activities</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="AliceBlue"> 
+      <h3 align="center">Education & Activities</h3>
+
+    <ul class="info" align="left">
+        <li><a href="https://github.com/kofimeighan">GitHub</a></li>
+        <li><a href="https://www.linkedin.com/in/kofimeighan/">LinkedIn</a></li>
+        <li>(561) 613-1936</li>
+        <li>kofimeighan@google.com</li>
+    </ul>
+
+    <ul class="nav">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="pictures.html">Film Pictures</a></li>
+        <li><a href="mixes.html">Music Mixes</a></li>
+    </ul>
+
+
+    <h4><u>Education</u></h4>
+    <p><i>Pine Crest School</i> - Pine Crest School is a private prepatory
+     school in Fort Lauderdale, Florida. I attended from August 2015 to 
```

Comment:
I think this section needs some formatting. Try moma "html style guide" internally.

Reply:
Just tried searching moma, I don't have access to the actual style guide.

Reply:
That is unfortunate... okay, I will try my best to read them and enforce..